Dependencies/napi/napi-jsi/CMakeLists.txt

```diff
@@ -13,7 +13,7 @@ add_library(napi ${SOURCES})
 target_include_directories(napi
     PUBLIC "include")
 
-target_link_to_dependencies(napi
+target_link_libraries(napi
```

Comment:
Was this change intentional?

Reply:
yes depending on make version target_link_to_dependencies doesn't exist as a valid cmake command. I'm not sure why we've been using it here.

Reply:
actually looks like this was a custom command. Do we have more context around why we've been using this custom command:

https://github.com/BabylonJS/BabylonNative/blob/master/Dependencies/CMakeExtensions/CMakeLists.txt

Reply:
I believe this was originally introduced, among other reasons, to copy DLLs for V8 on Windows. Does that still work with this change?

Reply:
I may be naive/wrong here, but it doesn't look like we are setting ON_LINKED_AS_DEPENDENCY_CMAKE_FILES for this library, so I don't think using the custom target_link_to_dependencies is necessary here?

Reply:
if folks want this reverted, I can look into updating our consuming projects to locate this target_link_to_dependencies function. That going said, if it doesn't serve any additional purpose it seems worth converting back to target_link_libraries so that external consumers of napi jsi don't have to include these additional cmake function definitions

Reply:
given this could potentially have issues on UWP but I've been primarily testing on iOS, I'm just going to refer this change.

Reply:
We will be using `add_on_linked_as_dependency_cmake_file` for napi-jsi in order to pull in v8jsi dll's (see #832). I also thought CMakeLists globally enforced a minimum version across files, so I'm not sure why iOS specifically would have issues consuming the custom function, unless there's a file with a version setter that needs to get revved.